components/event/EventCard.js

```diff
@@ -76,6 +76,18 @@ export default function EventCard({ event, username }) {
               <ReactMarkdown className="text-sm text-gray-500 py-1 flex-wrap">
                 {event.description}
               </ReactMarkdown>
+              <p className="text-sm text-gray-800 py-1 flex-wrap">
+                {event.isVirtual === true && event.isInPerson === false ? (<span>Remote</span>) : 
+                (<span>
+                  <span>{event.location.road}</span> 
```

Comment:
Instead of multiple spans, you can do something like this
<p>`${event.location.city && event.location.city + ","}  ${...}  ${...}  ${...}`</p>

all with a condition to show only is they are present

Reply:
Okay, got it. Will try now.

Reply:
We could also use `join`, I will change after merge